twitchio/abcs.py

```diff
@@ -159,18 +159,18 @@ async def clear(self):
 
         await ws.send_privmsg(channel, content=f'.clear')
 
-    async def slow(self):
+    async def slow(self, time: int=30):
```

Comment:
This should have whitespace around the =

Reply:
Whenever i looked at another method I viewed no white space for default values

Reply:
https://i.adiscorduser.com/kCDKS89.png

Reply:
Ok then 👍